<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptOptionsDialog</class>
 <widget class="QDialog" name="OptOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>156</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>156</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>156</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonNo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonYes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QSlider" name="threadAmountSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>64</number>
   </property>
   <property name="pageStep">
    <number>8</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Do you want to use multithreading?</string>
   </property>
  </widget>
  <widget class="QLabel" name="ThreadNumLabel">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
